backfill/2024-04-26-active_users_aggregates/fenix_query.sql

```diff
@@ -53,8 +73,6 @@ SELECT
   is_default_browser,
   COALESCE(REGEXP_EXTRACT(locale, r'^(.+?)-'), locale, NULL) AS locale,
   CASE
-    WHEN app_name = 'Fenix' AND isp = 'BrowserStack'
```

Comment:
Do we not need this anymore?

Reply:
Not for the backfill, it's already separated in unified_metrics.